grammar.js

```diff
@@ -70,8 +70,7 @@ const PREC = {
   // to assume they have default precedence.
   // https://github.com/wch/r-source/blob/0a8f53a7ba47463f1c938dd3e2c2acc7a2d3a1c2/src/main/gram.y#L467-L468
   // https://www.gnu.org/software/bison/manual/html_node/How-Precedence.html
-  // NOTE: See `NOTE ON PREC.RIGHT` above
-  BLOCK: { ASSOC: prec.right, RANK: 0 },
+  BLOCK: { ASSOC: prec, RANK: 0 },
```

Comment:
Note - can fix `prec.right()` -> `prec()` now that we don't have an optional `}` or `)`